ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -77,6 +77,12 @@ ref:
       * "" (default) - corresponds to "Optional" behaviour. 
       * "Required" - sets IMDSv2 to Required for all machines in install-config, metadata service is not accessible without auth procedure.
       * "Optional" - auth procedure on metadata service is optional
-
+  - name: PRESERVE_BOOTSTRAP_IGNITION
+    default: ""
+    documentation: |-
+      Allow users to make S3 deletion optional
+      More info:
```

Comment:
Maybe we can drop the "More info" to be more tidy.

Reply:
Updated.